src/Contexts/auth-context.jsx

```diff
@@ -11,12 +11,39 @@ const AuthProvider = ({ children }) => {
   const [token, setToken] = useState(localStorageItems?.userToken);
   const [user, setUser] = useState(localStorageItems?.activeUser);
   const loginCall = async (email, password) => {
+    console.log(email, password);
     try {
       const response = await axios.post("/api/auth/login", {
         email,
         password,
       });
 
+      if (response.status === 200 || response.status === 201) {
+        console.log("success", email, password);
+        localStorage.setItem(
+          "LoginCredentials",
+          JSON.stringify({
+            userToken: response.data.encodedToken,
+            activeUser: response.data.foundUser,
+          })
+        );
+        setToken(response.data.encodedToken);
+        setUser(response.data.foundUser);
+      }
+    } catch (error) {
+      console.log(error);
```

Comment:
For error messages, you can use `console.error`, but you can stick with this as well.